src/pulp/solvers.py

```diff
@@ -1230,9 +1230,11 @@ def actualResolve(self,lp):
 
 class XPRESS(LpSolver_CMD):
     """The XPRESS LP solver"""
+    csv = __import__('csv')
```

Comment:
I think the import statement should be passed to the top so it is explicit. At some point we will break the files into different files but for now it is good to have every import at the top.